components/link/link.js

```diff
@@ -139,13 +156,31 @@ class Link extends FocusMixin(LitElement) {
 			'truncate': this.lines > 0
 		};
 		const styles = (this.lines > 0) ? { '-webkit-line-clamp': this.lines } : {};
+		if (this.newWindow && this.target === undefined) {
+			this.target = '_blank';
+		}
 		return html`<a
 				aria-label="${ifDefined(this.ariaLabel)}"
 				class="${classMap(linkClasses)}"
 				style="${styleMap(styles)}"
 				?download="${this.download}"
 				href="${ifDefined(this.href)}"
-				target="${ifDefined(this.target)}"><slot></slot></a>`;
+				target="${ifDefined(this.target)}"><slot></slot>${this.renderNewWindowIcon()}</a>`;
+	}
+
+	renderNewWindowIcon() {
```

Comment:
Once the `classMap` is removed, this should be simple enough that I'd suggest just moving it right into `render()`.